<launch>
    <param name ="/use_sim_time" value="true"/>
    
    <rosparam command="load" file="$(find open3d_conversions_examples)/params/icp_bag_params.yaml" />
    
   <node name="tf_world_to_odom_init" pkg="tf" type="static_transform_publisher" args="0 0 0 -1.57079633 0 -1.57079633 world mapper_init 10"/>
<!-- <node name="tf_world_to_odom_init" pkg="tf" type="static_transform_publisher" args="-0.652977526188 1.58423674107 -1.2682980299 0.286120032948  -0.36926126894  0.505855208963 0.725184079784 ned mapper_init 10"/> -->
  

    <node name="perception_open3d_ex_icp" pkg="open3d_conversions_examples" type="ex_icp" output="screen">
    <!--    <remap from="pointcloud_in_topic" to="/transformed_cloud"/> -->
 <remap from="pointcloud_in_topic" to="/l515/depth/color/points"/>
<!-- <remap from="pointcloud_in_topic" to="/camera/depth/points2"/>  -->

        <!-- <remap from="pointcloud_out_topic" to="/transformed_cloud"/> -->
    </node>
</launch>
